snowflake_queries/snowflake_unusual_login_volume_query.yml

```diff
@@ -0,0 +1,23 @@
+AnalysisType: scheduled_query
+QueryName: Query.Snowflake.UnusualLoginVolume
+Enabled: false
+Description: >
+  Alert on a high number of logins (above a baselined threshold)
+AthenaQuery: >
+  /* athena query not supported */
+  SELECT count(1)
+SnowflakeQuery: >
+
+  SELECT DISTINCT user_name, count(user_name), first_authentication_factor
+  FROM snowflake.account_usage.login_history
+  WHERE p_occurs_since('10 h' ,, event_timestamp)
```

Comment:
looks like the rate minutes specify this should run 1/day, but we only look at the last 10 hours of activity here.

Reply:
Thanks for catching that! Remnant of me testing the query then pasting it back into the YAML